pkg/kudoctl/cmd/install/install.go

```diff
@@ -33,9 +33,18 @@ var DefaultOptions = &Options{
 // Run returns the errors associated with cmd env
 func Run(cmd *cobra.Command, args []string, options *Options) error {
 
-	// This makes --kubeconfig flag optional
-	if _, err := cmd.Flags().GetString("kubeconfig"); err != nil {
-		return fmt.Errorf("get flag: %+v", err)
+	err := validate(args, options)
+	if err != nil {
+		return err
+	}
+
+	err = installOperator(args[0], options)
+	return err
+}
+
+func validate(args []string, options *Options) error {
+	if len(args) != 1 {
+		return fmt.Errorf("no argument provided, need name of the package or path to install")
```

Comment:
```
$ go run cmd/kubectl-kudo/main.go install zk zookeeper kafka
Error: no argument provided, need name of the package or path to install
exit status 255
```

We should make the error message more precise.

```suggestion
		return fmt.Errorf("too many or no argument(s) provided, need name of the package or path to install")
``` 

?